src/client/hooks/useChapters.ts

```diff
@@ -104,6 +105,9 @@ function isChapterActive(
 
     case Chapters.KVK:
       return !isLoading(KVK) && !!KVK.content;
+
+    case Chapters.TOERISME:
+      return !isLoading(TOERISME) && !!TOERISME.content;
```

Comment:
Als we een array gebruiken voor de respone data kan die mogelijk leeg zijn. Hier moeten we dus op de `.length` variabele checken.